pkg/search/postgres/common.go

```diff
@@ -1005,16 +1005,15 @@ func RunGetManyQueryForSchema[T any, PT pgutils.Unmarshaler[T]](ctx context.Cont
 
 func prepareQuery(ctx context.Context, schema *walker.Schema, q *v1.Query) (*query, error) {
 	if q == nil {
+		log.Info("query is nil, using empty")
 		q = searchPkg.EmptyQuery()
 	}
 
 	preparedQuery, err := standardizeQueryAndPopulatePath(ctx, q, schema, GET)
 	if err != nil {
+
 		return nil, errors.Wrap(err, "error creating query")
 	}
-	if preparedQuery == nil {
```

Comment:
This was causing an issue in a test.  The search path was slightly different than the `GetByQueryFn` path.  in the search path this simply returns no results whereas the `GetByQuery` path was throwing an error.  IMO that an empty result set is the desired behavior in the `GetByQuery` path as well.  It prevents us from accidentally exposing implementation details that really there is no fundamental difference.